tutorials/text/mathtext.py

```diff
@@ -76,10 +76,10 @@
 
     \sum_{i=0}^\infty x_i
 
-Fractions, binomials and stacked numbers
-----------------------------------------
+Fractions, binomials, and stacked numbers
+-----------------------------------------
 
-Fractions, binomials and stacked numbers can be created with the ``\frac{}{}``,
+Fractions, binomials, and stacked numbers can be created with the ``\frac{}{}``,
```

Comment:
Line too long with the comma 😄

Reply:
I'm now wondering if I should use the github interface or give up on being lazy and clone the code…

Reply:
decision postponed after the coffee break :coffee:

Reply:
fixed